components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.geo-dashboard/public/js/geo_fencing.js

```diff
@@ -431,3 +431,76 @@ function viewFence(geoFenceElement,id) {
     }
     closeAll();
 }
+
+function viewFenceByData(geoJson, queryName, areaName, stationeryTime, id) {
+    var matchResults = /(?:"geoFenceGeoJSON"):"{(.*)}"/g.exec(geoJson);
+    if (matchResults && matchResults.length > 1) {
+        geoJson = "{" + matchResults[1] + "}";
+    }
+    geoJson = JSON.parse(geoJson.replace(/'/g, '"'));
+    var geometryShape;
+
+    if(geoJson.type=="Point"){
+
+        var circleOptions = {
+            color: '#ff0043'
+        };
+        geometryShape= new L.circle([geoJson.coordinates[1],geoJson.coordinates[0]], geoJson.radius,circleOptions);
+        // var marker=new L.marker([geoJson.coordinates[1],geoJson.coordinates[0]]);
+        map.addLayer(geometryShape);
+        //  map.addLayer(marker);
+    } else if(geoJson.type=="Polygon"){
```

Comment:
Fixed the formatting issues in https://github.com/wso2/carbon-device-mgt/pull/950